doc/config.example.yaml

```diff
@@ -37,15 +31,27 @@ node:
   max_retries: 5
   # Timeout for each request in milliseconds
   timeout: 2500
+  # Path to the data directory (if the path doesn't exist it will be created)
+  data_dir: .cache
+
+################################################################################
+##                             Validator configuration                        ##
+## The server can operate in two modes: full node and validator node.         ##
+## The full node does not participate in consensus,                           ##
+## it only replicates the state of the blockchain.                            ##
+################################################################################
+validator:
+  # Whether or not we should act as a validator
+  # When validating, the `seed` of an eligible account is required
+  # An eligible account has at least 4 units frozen in it
+  enabled: true
   # This is a randomly generated keypair
   # If this node is not a validator, this will be ignored
   #
   # DO NOT USE THOSE VALUES ANYWHERE
   # Private seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
   # Public address:  GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ
   seed:    SCT4KKJNYLTQO4TVDPVJQZEONTVVW66YLRWAINWI3FZDY7U4JS4JJEI4
-  # Path to the data directory (if the path doesn't exist it will be created)
-  data_dir: .cache
```

Comment:
Unrelated change? The FullNode also needs a cache directory.

Reply:
For some reason, the `git diff` deleted it here and moved it to line 34-35.

Reply:
Oh I see!